citus_dev/citus_dev

```diff
@@ -188,7 +201,7 @@ def main(arguments):
         clustername = arguments["<name>"]
         port = int(arguments["--port"])
         for role in getRoles(clustername):
-            run(f'pg_ctl {pgctl_flags} start -D {clustername}/{role} -o "-p {port}" -l {role}_logfile')
+            run(f'pg_ctl {pgctl_flags} start -D {clustername}/{role} -o "-p {port}"')
```

Comment:
can the port be removed if we have it in the config now?

possibly we would also need to remove the flag from start/restart, given it's a property of the cluster now, not a runtime argument.